apps/client/src/queries/code/useGetLocation.get.tsx

```diff
@@ -0,0 +1,10 @@
+import { useQuery } from '@tanstack/react-query';
+import { codeAPI, CodeResDto } from '@oseek/apis';
+
+export const useGetLocation = () => {
```

Comment:
```suggestion
export const useGetLocation = () => useQuery({
  queryKey: ['jeju_location'],
  queryFn: () => codeAPI.findCodeAxios('jeju_location'),
  select: ({data}) => data?.data,
  enabled: ...
});
```
이런 식으로 하시면 왠지 타입도 알아서 추론이 될 거 같아요 ㅎㅎ
그리고, 좀 더 편하게 Option들을 넣을 수 있을거 같아요! 참고만 해주세요 :) 
제가 아직 해보지는 않았는데...

Reply:
react query 옵션 관련해서도 조사해봐야겠네요.
리뷰 감사합니다!

다음 개선 작업해보면서 반영해볼게요 ㅎㅎ